cerbero/bootstrap/site-patch.py

```diff
@@ -25,7 +25,7 @@ def __boot():
                 break
         else:
             try:
-                import imp  # Avoid import loop in Python 3
+                import importlib as imp  # Avoid import loop in Python 3
```

Comment:
This is not the right solution:

`AttributeError: module 'importlib' has no attribute 'find_module'`